SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -270,6 +281,9 @@ bool FSpatialGDKEditorModule::ForEveryServerWorker(TFunction<void(const FName&,
 
 FPlayInEditorSettingsOverride FSpatialGDKEditorModule::GetPlayInEditorSettingsOverrideForTesting(UWorld* World) const
 {
+	// By default, clear that it was loaded from taken snapshot.
```

Comment:
// By default, clear that the runtime/test was loaded from a snapshot taken for a given world.